dc_utils/context_processors.py

```diff
@@ -1,9 +1,11 @@
 from django.conf import settings
+import django
 
 
 def dc_django_utils(request):
     return {
         "SITE_LOGO_WIDTH": "100",
         "SITE_TITLE": getattr(settings, "SITE_TITLE", ""),
         "CANONICAL_URL": f"{request.scheme}://{request.get_host()}",
+        "VERSION": getattr(settings, "VERSION", django.VERSION[0]),
```

Comment:
Might be useful if this were called `DJANGO_MAJOR_VERSION` so we know what version we're talking about in the templates